### DISCUSSION TITLE
DEFAULT_SSID_NOT_SET

### DISCUSSION THREAD

**Discussion by jbreske** [forkineye/ESPixelStick#676] [Q&A]
Two new Dig Quad AE+.  Flash them just I have any other.  Flashing with build 4.0-ci6222391064.  Everything seems to flash fine.  The ESP attempts to connect via wifi and I consistently get "DEFAULT_SSID_NOT_SET" and wifi connection failures.  I have four other Dig Quad AE+ flashed and running successfully.  Thanks for the assistance.

**Comment by jbreske** [comment]
Log attached. [Default SSID.txt](https://github.com/forkineye/ESPixelStick/files/12901425/Default.SSID.txt)

**Comment by MartinMueller2003** [comment]
That message comes out when the attempt to use the configured credentials fails and the State Machine tries to fall back to the credentials set at compile time (which dont exist). Did you verify the credentials in the log?

You have AP fallback turned off. I would turn it on and connect to the device using a tablet etc and verify the WiFi settings via the UI.

**Reply by jbreske** [reply]
I did notice the AP fallback turned off but don't actually know what that means. Past devices flashed fine and then I get the DHCP IP address assigned. Then by the IP or SSID I could fine them in my router to give a static assignment. These two devices just kept recycling and attempting to connect via wifi. I could never see them in my wiifi router.  I'm actually not sure how to access the UI without knowing the IP address. I'll try the AP fallback and see what happens. Thanks for the assistance.

**Comment by MartinMueller2003** [comment]
Reflash. Change the AP reboot to off so it stops rebooting if it cant connect. Set AP fallback ON. It will go into AP fallback and become a nearby AP you can connect to.  Open a tablet or phone, go to your WiFi settings, select the ESP network. Go to the ESP UI at the address you see on the console output.

**Reply by jbreske** [reply]
That got me in as an AP. Not sure why it won't connect to wifi. The network setup tab has the proper credentials entered for accessing my network.

**Reply by jbreske** [reply]
Apologies for wasting your time but thanks for some education.  The AE+ has an external antenna...which I didn't have connected. Little to no reception range without the antenna.

**Comment by MartinMueller2003** [comment]
No worries. FYI: Operating without the antenna can damage the RF driver. Lets hope they are ok.
